src/Microsoft.Performance.Toolkit.Engine.Tests/ProcessingOptionsResolverTests.cs

```diff
@@ -0,0 +1,254 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.SDK.Processing;
+using Microsoft.Performance.SDK.Processing.DataSourceGrouping;
+using Microsoft.Performance.SDK.Processing.Options;
+using Microsoft.Performance.SDK.Runtime;
+using Microsoft.Performance.Testing;
+using Microsoft.Performance.Toolkit.Engine;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source123;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source4;
+using Microsoft.Performance.Toolkit.Engine.Tests.TestCookers.Source5;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Performance.SDK.Tests
+{
+    [TestClass]
+    public class ProcessingOptionsResolverTests
+    {
+        [TestMethod]
+        [UnitTest]
+        public void DefaultGlobalProcessingOptionsResolverTest()
+        {
+            var sut = new GlobalProcessingOptionsResolver(ProcessorOptions.Default);
+
+            Assert.IsNotNull(sut, "Options Resolver is null");
+
+            var dsg1 = new DataSourceGroup(
```

Comment:
There is a method `Any.DataSourceGroup()` you can call to quickly create these instances.

Reply:
If I were to use Any.DataSourceGroup it looks like it would always create the same dsg. Which is not great for all tests.